src/ui/zabapgit_js_common.w3mi.data.js

```diff
@@ -1666,14 +1666,22 @@ Hotkeys.prototype.showHotkeys = function() {
   }
 };
 
-Hotkeys.prototype.getAllSapEventsForSapEventName = function(sSapEvent) {
-  return [].slice.call(
-    document.querySelectorAll('a[href*="sapevent:' + sSapEvent + '"],'
-      + 'a[href*="SAPEVENT:' + sSapEvent + '"],'
-      + 'input[formaction*="sapevent:' + sSapEvent + '"],'
-      + 'input[formaction*="SAPEVENT:' + sSapEvent + '"],'
-      + 'form[action*="sapevent:' + sSapEvent + '"] input[type="submit"].main,'
-      + 'form[action*="SAPEVENT:' + sSapEvent + '"] input[type="submit"].main'));
```

Comment:
Looks like you dropped `input type=submit` but I guess there's no hotkey for the default action (i.e. it's always `enter`.

Reply:
Yes that was intentional. All hotkeys mentioned in the original PR also worked without.